tests/console/ping.pm

```diff
@@ -21,20 +21,24 @@ use version_utils qw(is_jeos is_sle);
 
 sub run {
     my ($self) = @_;
+    my $ping_group_range = script_output('sysctl net.ipv4.ping_group_range');
```

Comment:
This can be executed in user console depending on what has been left by previous test. e.g. https://openqa.suse.de/tests/10266123#step/ping/3

I guess it should go after `select_serial_terminal;`

Reply:
Thanks! The sysctl value should be the same whatever user executes it, but it would fail in case this is the first test. I'll fix it.

Reply:
I have moved the code after selecting the console. Waiting for VR afterwards I can push the code. I will ping you

Reply:
PR to fix this: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16172